src/api/app/models/service/name_validator.rb

```diff
@@ -15,11 +15,15 @@ def valid?
     def valid_name?
       return false unless name.is_a?(String)
```

Comment:
maybe we should invert both lines? before we check if it's `name.blank? || name.size > 200` and then we check the class?

Reply:
I would replace this type check with a `respond_to?(:blank?)`

Reply:
> I would replace this type check with a `respond_to?(:blank?)`

I'm not sure, why?

```
[1] pry(main)> nil.blank?
=> true
```

Reply:
I'm talking about the `unless name.is_a?(String)`.
We use `duck typing` we don't really care if its really a string or not...
Do you agree?

Reply:
but everything answers to blank no?

```
[2] pry(main)> 1.respond_to?(:blank?)
=> true
```

Reply:
IMO @danidoni they check if it's string to avoid any XML injection attack.

Reply:
That makes sense